maths/proth_number.py

```diff
@@ -58,6 +58,12 @@ def proth(number: int) -> int:
 
 
 if __name__ == "__main__":
-    import doctest
+    for number in range(11):
```

Comment:
Let's test the code first and then let the user interact with it.

Best of both worlds!
```suggestion
    import doctest
    
    doctest.testmod()
    for number in range(11):
```